test/jdk/java/net/DatagramSocket/UnreferencedDatagramSockets.java

```diff
@@ -130,6 +137,7 @@ public static void main(String args[]) throws Exception {
 
         p = new DatagramPacket(msg, msg.length);
         client.receive(p);
+        svr.latch.countDown(); // unblock the server
 
```

Comment:
Please add a second empty line as does the original change.

Reply:
fixed